docs/userguide/data_buildtool.md

```diff
@@ -0,0 +1,372 @@
+
+    localeFilter: {
+      filterType: locale
+      whitelist: [
+        en
+        de
+        zh
+      ]
+    }
+
+#### Adding Script Variants (includeScripts = true)
+
+You may set the *includeScripts* option to true to include all scripts for a language while using *filterType* "locale".  This results in behavior similar to *filterType* "language".  In the following JSON example, all scripts for Chinese are included:
+
+    {
+      "localeFilter": {
+        "filterType": "locale",
+        "includeScripts": true,
+        "whitelist": [
+          "en",
+          "de",
+          "zh"
+        ]
+      }
+    }
+
+If you wish to explicitly list the scripts, you may put the script code in the locale tag in the whitelist, and you do not need the *includeScripts* option enabled.  For example, in Hjson, to include Han Traditional but not Han Simplified:
+
+    localeFilter: {
+      filterType: locale
+      whitelist: [
+        en
+        de
+        zh_Hant
+      ]
+    }
+
+Note: the option *includeScripts* is only supported at the language level; i.e., in order to include all scripts for a particular language, you must specify the language alone, without a region tag.
+
+#### Removing Regional Variants (includeChildren = false)
+
+If you wish to enumerate exactly which regional variants you wish to support, you may use *filterType* "locale" with the *includeChildren* setting turned to false.  The following *filters.hjson* file includes English (US), English (UK), German (Germany), and Chinese (China, Han Simplified), as well as their dependencies, *but not* other regional variants like English (Australia), German (Switzerland), or Chinese (Taiwan, Han Traditional):
+
+    localeFilter: {
+      filterType: locale
+      includeChildren: false
+      whitelist: [
+        en_US
+        en_GB
+        de_DE
+        zh_CN
+      ]
+    }
+
+### File Slicing (coarse-grained features)
+
+ICU provides a lot of features, of which you probably need only a small subset for your application.  Feature slicing is a powerful way to prune out data for any features you are not using.
+
+***CAUTION:*** When slicing by features, you must manually include all dependencies.  For example, if you are formatting dates, you must include not only the date formatting data but also the number formatting data, since dates contain numbers.  Expect to spend a fair bit of time debugging your feature filter to get it to work the way you expect it to.
+
+The data for many ICU features live in individual files.  ICU Data Build Tool puts similar *types* of files into categories.  The following table summarizes the ICU data files and their corresponding features and categories:
+
+Feature | Category ID(s) | Data Files <br/> ([icu4c/source/data](https://github.com/unicode-org/icu/tree/master/icu4c/source/data))
```

Comment:
More of a minor nit: the markdown table seems to render a bit mangled on GitHub.
(Was it exported from something perhaps?)

Reply:
I hand-wrote the table. It renders correctly in my local Markdown renderer (non-GFM).  I'll look into it.

Reply:
I fixed the table for GitHub and added a column showing file sizes.